src/transformers/models/roformer/tokenization_roformer.py

```diff
@@ -66,6 +66,236 @@
 }
 
 
+# copied from ..bert.tokenization_bert
```

Comment:
```suggestion
# Copied from transformers.models.bert.tokenization_bert.load_vocab
```
The copied from statement needs to begin with a capital and use the full absolute path. Showed here above, it's the same for all other copied from in this file.